src/main/java/hudson/plugins/git/util/DefaultBuildChooser.java

```diff
@@ -120,7 +120,7 @@ public DefaultBuildChooser() {
                 verbose(listener, "{0} seems to be a non-branch reference (tag?)");
             }
         }
-        
```

Comment:
This whitespace changes disturb every contributor. Could you provide link to such discussion?